src/MultiSelect.php

```diff
@@ -94,6 +106,9 @@ protected function fillAttributeFromRequest(NovaRequest $request, $requestAttrib
         if ($request->input($requestAttribute)) {
             $requestIds = collect(explode(',', $request->input($requestAttribute)));
         }
+
+        $model->save();
```

Comment:
What does this save call accomplish?

Reply:
When we create a new record it first writes to the the pivot table.  The pivot table is then missing the id from the record because it doesn't exist yet.  Telling the model to save writes the record so the id exists.